draft-id-claim-169-v1-2-0.md

```
@@ -1,54 +1,58 @@
 # 169 - QR Code Specifications
 
-## CBOR Identity Data in QR Code
+### CBOR Identity Data in QR Code
```

markdownlint-cli2 (0.18.1)

Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)